pkg/config/configsections/request.go

```diff
@@ -35,3 +35,10 @@ type CertifiedOperatorRequestInfo struct {
 	// Organization as understood by the operator publisher , e.g. `redhat-marketplace`
 	Organization string `yaml:"organization" json:"organization"`
 }
+
+// AcceptedKernelTaintsRequestInfo contains all certified operator request info
+type AcceptedKernelTaintsRequestInfo struct {
```

Comment:
The other two were called something-requestinfo, because it's used to form a request to catalog api. This one probably can just be AcceptedKernelTaints or AcceptedKernelTaintsInfo?